Packages/MIES/MIES_SweepFormula.ipf

```diff
@@ -3268,6 +3268,27 @@ static Function/WAVE SF_OperationEpochsImpl(string graph, WAVE/T epochPatterns,
 	return output
 End
 
+static Function SF_AssertOnMismatchedWaves(WAVE data0, WAVE data1, string opShort)
+
+	string msg, size0Str, size1Str
+	variable ret
+
+	ret = EqualWaves(data0, data1, EQWAVES_DIMSIZE)
+
+	if(ret)
+		return NaN
+	endif
+
+	WAVE size0 = GetWaveDimensions(data0)
+	WAVE size1 = GetWaveDimensions(data1)
+
+	size0Str = NumericWaveToList(size0, ", ", trailSep = 0)
+	size1Str = NumericWaveToList(size1, ", ", trailSep = 0)
+	sprintf msg, "%s: wave size mismatch [%s] vs [%s]", opShort, size0Str, size1Str
```

Comment:
Would it be possible for the error message to report the line number in SF with the error?